src/app/api/ogImages/route.tsx

```diff
@@ -1,127 +1,89 @@
-import * as fs from 'node:fs/promises'
-import { resolve } from 'node:path'
-
 import { ImageResponse } from 'next/og'
-import { notFound } from 'next/navigation'
-import type { NextRequest } from 'next/server'
-
-import prisma from '@db/prismaSingleton'
-
-import NonInteractiveBench from '@components/NonInteractiveBench'
-import NonInteractiveField from '@components/NonInteractiveField'
-
-export const GET = async (request: NextRequest) => {
-  try {
-    const teamSheetId = request.nextUrl.searchParams.get('teamSheetId')
-
-    const teamSheet = teamSheetId
-      ? await prisma.teamSheet.findUnique({
-        where: {
-          shareId: teamSheetId,
-        },
-      })
-      : null
 
-    if (!teamSheet) return notFound()
+export const runtime = 'edge'
```

Comment:
Was trying to understand what edge run time does, Is it just meant to increase the speed of the ogImages by using local servers?

[https://nextjs.org/docs/app/api-reference/file-conventions/metadata/opengraph-image](url)

This is an article I was reading about edge, is there a better one?
[https://medium.com/@dawid.niegrebecki/why-you-should-deploy-nextjs-on-the-edge-cbfc304b58c](url)

article goes on to say it makes no difference unless you have databases in different regions as well